frontend/src/App.js

```diff
@@ -1,23 +1,53 @@
-import logo from './logo.svg';
-import './App.css';
+import {useEffect, useReducer, useState} from 'react'
+import countriesReducer from './reducer/countriesReducer'
+
+import Header from './components/Header'
+import Footer from './components/Footer'
+import CountryDropdowns from './components/CountryDropdowns'
+import SymmetricCountryChart from './components/SymmetricCountryChart'
+import GlobalStyles from "./GlobalStyles";
 
 function App() {
+
+  const [countries, setCountries] = useReducer(
+    countriesReducer,
+    {data: [], isLoading: false, isError: false}
+  )
+  
+  const [displayedCountries, setDisplayedCountries] = useState({
+    countryLeft: {},
+    countryRight: {}
+  })
+  
+  useEffect(() => {
+    setCountries({type: 'COUNTRIES_FETCH_INIT'})
+    fetch('http://countrycheck.local/countries')
+      .then(response => response.json())
+      .then(result => {
+          setCountries({
+          type: 'COUNTRIES_FETCH_SUCCESS',
+          payload: result
+          })
+      })
+      .catch(() => setCountries({type: 'COUNTRIES_FETCH_FAILURE'}))
+  }, [])
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <GlobalStyles/>
+      <Header/>
+     {countries.isError ?? <p>An error occurred while fetching data</p>}
```

Comment:
Be careful using `??` here because it will never return your error message. 
The only case where you would see your paragraph is, when `isError` is not defined.

```suggestion
     {countries.isError && <p>An error occurred while fetching data</p>}
```

See https://javascript.info/nullish-coalescing-operator

Reply:
Right! What a stupid mistake 👍